### PR TITLE
Issue 228, fix header styles in vms

### PR DESCRIPTION
**Pull request by cewing**
This pr completes the job of making the header styles in VMS match the finalized designs.

It also uses the defaultUser image for authenticated users who do not have a portrait set, which is not in the designs specifically, but which matches the Plone site header.  

This PR must be combined with washingtontrails/wta_buildout#19, which implements passing the user portrait URL in the SSO token.  This implementation is safe in that it will not crash if that has not yet been released, but user portraits will not appear until it is.

### CODE REVIEW

**Comment by davisagli** [comment]
Where's the wta_buildout PR?

**Comment by cewing** [comment]
there.  Had to make one first

File: src/vms/static/app/nav.js [hunk 1]
```diff
@@ -2,6 +2,40 @@ var React = require('react');
...
+		// 		  <div className="portrait">
+		// 			&nbsp;
+		// 		  </div>My Backpack</a>
+		// 	);
+		// } else {
+		// 	return (
+		// 		<a className="icon" href={url + '/backpack'}>
+		// 		  <div className="portrait">&nbsp;</div>My Backpack</a>
+		// 	);
+		// }
```
Comment (davisagli) [inline src/vms/static/app/nav.js]: no commented code, please

Comment (cewing) [inline src/vms/static/app/nav.js]: ack. sorry

**Pull request merged**